package.json

```diff
@@ -16,8 +16,10 @@
   "scripts": {
     "lint": "eslint .",
     "lint-fix": "eslint . --fix",
-    "test": "npm install --no-save \"gl@^6.0.2\" && npm run lint && jest",
-    "jest": "jest",
+    "test": "npm install --no-save \"gl@^6.0.2\" && npm run lint && npm run vitest",
+    "test:vitest": "vitest",
+    "test:vitest:watch": "vitest --watch",
+    "test:vitest:coverage": "vitest run --coverage --coverage.reporter=text-summary --coverage.dir=./coverage",
```

Comment:
Remove the `test:` prefix or fix the `test` command (it only says `vitest`, not `test:vitest`)